sys-apps/fwupd/fwupd-1.3.2.ebuild

```diff
@@ -5,30 +5,45 @@ EAPI=7
 
 PYTHON_COMPAT=( python3_{5,6,7} )
 
-inherit meson python-single-r1 vala xdg-utils
+inherit linux-info meson python-single-r1 vala xdg
 
 DESCRIPTION="Aims to make updating firmware on Linux automatic, safe and reliable"
 HOMEPAGE="https://fwupd.org"
 SRC_URI="https://github.com/hughsie/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-2.1+"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~x86"
-IUSE="agent colorhug consolekit dell doc elogind +gpg +man nvme pkcs7 redfish systemd test thunderbolt uefi"
-
-REQUIRED_USE="${PYTHON_REQUIRED_USE}
+KEYWORDS="~amd64"
```

Comment:
There's a better way, you can mask >=fwupd-1.3.2 on arm and x86 while mentioning the keywording bug in the mask. When the dep is keyworded, the mask can just be lifted instead of having to re-keyword this package (its slow).

Reply:
I think I got the masks working right :)